tests/test_models.py

```diff
@@ -18,14 +18,12 @@
 
 import pykeen.experiments
 import pykeen.models
-from pykeen.datasets.kinships import TRAIN_PATH as KINSHIPS_TRAIN_PATH
+from pykeen.datasets.kinships import KINSHIPS_TRAIN_PATH as KINSHIPS_TRAIN_PATH
```

Comment:
redundant `import KINSHIPS_TRAIN_PATH as KINSHIPS_TRAIN_PATH`

Reply:
oh solid. resolved in c2c3fa26